index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <style>
+        body {
+            background-image: url("Project materials/cover.png");
+            height: 100%;
+            width: 100%;
+            background-position: center;
+            background-repeat: no-repeat;
+            background-size: cover;
+            position: relative;
+        }
+
+        body::before {
+            content: '';
+            display: block;
+            width: 100%;
+            height: 100%;
+            position: absolute;
+            top: 0;
+            left: 0;
+            background-color: rgba(0, 0, 0, 0.5);
+            z-index: -1;
+        }
+
+        .container {
+            display: flex;
+            max-width: 1050px;
+            padding: 10%;
+            flex-direction: column;
+            align-items: center;
+            gap: 40px;
+            margin: auto;
+        }
+
+        h1 {
+            color: white;
+            text-align: center;
+            display: block;
+            font-family:'Times New Roman', Times, serif;
+            font-size: 5rem;
+            font-style: normal;
+            font-weight: 50rem;
+            line-height: 80px;
+            letter-spacing: 0.2px;
+            width: 573px;
+            height: 143px;
+        }
+
+        h4 {
+            color: white;
+            text-align: center;
+            font-family:'Times New Roman', Times, serif;
+            font-size: 1em;
+            font-style: normal;
+            width: 31em;
+            line-height: 30px;
+            letter-spacing: 0.2px;
+        }
+        
+        .topbar {
+            display: flex;
+            flex-direction: row;
+        }
+
+        .logo {
+            color: white;
+            padding: 14px 16px;
+            padding-right: 462px;
+            text-decoration: none;
+            font-size: 17px;
+            padding-left: 10%;
+        }
+
+        .logo:hover {
+            color: grey;
+        }
+
+        .menu {
+            float: right;
+            display:flex;
+            padding-right: 10%;
+        }
+
+        .menu a {
+            float: left;
+            display: flex;
+            color: white;
+            padding: 14px 16px;
+            text-decoration: none;
+            font-size: 17px;
+            gap: 20px;
+        }
+
+        .menu a:hover {
+            color: grey;
+        }
+
+        .imagemenu {
+            float: right;
+            display:flex;
+            padding-right: 10%;
+        }
+        .imagemenu a {
+            float: right;
```

Comment:
why do you add `float` property? 
The parent is flex, so your elements are already in 1 line.
float here is redundant. 

This comment relates to all cases where you use float

Reply:
Thank you for your feedback and for the tips to improve my code.